1st commit

RedisculuS opened the pull request:
added css for white theme for 3 devices

RedisculuS committed:
1st commit

added css for white theme for 3 devices

RedisculuS commented:
ще нема темної теми та фонове зображення тримає з мене прикола

anastasiiatarasiuk reviewed (changes requested):
Потрібно правильно завантажити фонові зображення, а також виправити стилі в списку соц мереж

RedisculuS committed:
2nd commit

changes according to team-lead review
changing backgrounds for tablet and desktop versions

anastasiiatarasiuk reviewed (changes requested):
У медіаправилі для десктопа в класі social-list потрібно забрати width: 1280px;

RedisculuS committed:
3rd commit

changes done

anastasiiatarasiuk reviewed (changes requested):
потрібно зробити фон списку з посиланнями по ширині сторінки

RedisculuS committed:
4th commit

тепер фон на всю сторінку

anastasiiatarasiuk reviewed (changes requested):
Прибрати від'ємні маржини та змінити маржин для заголовку з 112px на 88px

RedisculuS committed:
5th commit

Changes done

anastasiiatarasiuk reviewed (changes requested):
Для заголовка потрібно забрати margin-top. Але для класу hero-section потрібно поставити padding-top. у мобілки padding-top повинен бути 96px, для планшета 88px, для десктопа 168px. Це потрібно для того щоб background-image не виходив зя межі секції

RedisculuS committed:
6th commit

changes done
padding-top successfully added to hero-section
margin-top deleted from hero-title

anastasiiatarasiuk reviewed (changes requested):
Потрібно додати в тег a  з класом order-project посилання на секцію work together -  href="#contacts"

RedisculuS committed:
7th commit

 href="#contacts" successfully added

anastasiiatarasiuk reviewed (changes requested):
Потрібно видалити зайві фонові зображення

RedisculuS committed:
8th commit

Background pictures, that not used, are deleted

RedisculuS committed:
9th commit

Background pictures names are changed
